css/main.css

```diff
@@ -325,3 +325,31 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+form {
+  margin: auto;
+  margin-top: 10px;
+  display: flex;
+  flex-direction: column;
+}
+
+input {
+  border: none;
+  border-bottom: 1px solid rgb(155, 155, 155);
+  padding-bottom: 10px;
```

Comment:
- [ ] I will suggest you create a class for `padding-bottom` and `border`  property and then reuse it in your html. Since it's being repeated more than once in your css codes. This is to ensure DRY. E.g
```CSS
.padding-border {
  border: none;
  padding-bottom: 10px;
}
You can then add this class to the respective html
```